src/v2/options.ts

```diff
@@ -215,6 +215,11 @@ export interface GlobalOptions {
    * Invoker to set access control on https functions.
    */
   invoker?: 'public' | 'private' | string | string[];
+
+  /*
+   * Secrets to bind to a functions.
+   */
+  secrets?: string[];
```

Comment:
Can secrets be set globally (since this type is called `GlobalOptions`)?

Reply:
Yes, though it's a good way to cause a minor bill increase and lower security.

Reply:
Yeah, we should definitely encourage per-function. Interesting to know that it's possible globally though, thanks!